cli/config/configuration.toml.in

```diff
@@ -27,6 +27,10 @@ machine_type = "@MACHINETYPE@"
 # may stop the virtual machine from booting.
 # To see the list of default parameters, enable hypervisor debug, create a
 # container and look for 'default-kernel-parameters' log entries.
+#
+# NOTE: If you want to use systemd or any other init daemon, the `init` parameter
+# should be used here. For example for systemd you have to add following line:
+# init=/lib/systemd/systemd systemd.unit=@PROJECT_TAG@.target systemd.mask=systemd-networkd.service systemd.mask=systemd-networkd.socket
```

Comment:
I think we need to be a little careful about the wording here as we will be using systemd by default.

Also, since we're removing the systemd masks here, shouldn't they be added to the osbuilder PR (or maybe we no longer need them at all)?

Reply:
@jodh-intel 

> Also, since we're removing the systemd masks here, shouldn't they be added to the osbuilder PR (or maybe we no longer need them at all)?

we don't need them, these mask files are present if systemd is installed (AGENT_INIT=no in osbuilder)